src/benchmarklib/tpcc/tpcc_table_generator.cpp

```diff
@@ -22,9 +22,11 @@
 
 namespace opossum {
 
-TpccTableGenerator::TpccTableGenerator(const ChunkOffset chunk_size, const size_t warehouse_size)
+TpccTableGenerator::TpccTableGenerator(const ChunkOffset chunk_size, const size_t warehouse_size,
+   const bool encode_chunks)
```

Comment:
Encode... with what? Please chat with @lawben to make this consistent across all benchmarks.

Reply:
I would suggest passing an `std::optional<EncodingConfig> encoding_config` which then automatically specifies the encoding for all tables/columns if the user wants this. But that requires #906 to be merged first. 

If this needs to be merged ASAP, I would suggest it be merged (because the way it is now is also kinda not consistent) and I will work on this Wednesday/Thursday. The changes required are rather minimal.

Reply:
It does not need to be merge ASAP. Lets wait for #906.

Reply:
#906 is now merged.